src/pages/Explore/index.tsx

```diff
@@ -0,0 +1,227 @@
+import { Trans } from '@lingui/macro'
+import { BrowserEvent, InterfaceElementName, InterfacePageName, SharedEventName } from '@uniswap/analytics-events'
+import { TraceEvent } from 'analytics'
+import { Trace } from 'analytics'
+import { AutoRow } from 'components/Row'
+import {
+  LARGE_MEDIA_BREAKPOINT,
+  MAX_WIDTH_MEDIA_BREAKPOINT,
+  MEDIUM_MEDIA_BREAKPOINT,
+} from 'components/Tokens/constants'
+import { filterStringAtom } from 'components/Tokens/state'
+import NetworkFilter from 'components/Tokens/TokenTable/NetworkFilter'
+import SearchBar from 'components/Tokens/TokenTable/SearchBar'
+import TimeSelector from 'components/Tokens/TokenTable/TimeSelector'
+import TokenTable from 'components/Tokens/TokenTable/TokenTable'
+import { MouseoverTooltip } from 'components/Tooltip'
+import { useInfoExplorePageEnabled } from 'featureFlags/flags/infoExplore'
+import { useResetAtom } from 'jotai/utils'
+import { useEffect, useState } from 'react'
+import { useLocation } from 'react-router-dom'
+import styled, { css } from 'styled-components'
+import { ThemedText } from 'theme/components'
+
+const ExploreContainer = styled.div`
+  width: 100%;
+  min-width: 320px;
+  padding: 68px 12px 0px;
+
+  @media only screen and (max-width: ${({ theme }) => `${theme.breakpoint.md}px`}) {
+    padding-top: 48px;
+  }
+
+  @media only screen and (max-width: ${({ theme }) => `${theme.breakpoint.sm}px`}) {
+    padding-top: 20px;
+  }
+`
+const TitleContainer = styled.div`
+  margin-bottom: 32px;
+  max-width: ${MAX_WIDTH_MEDIA_BREAKPOINT};
+  margin-left: auto;
+  margin-right: auto;
+  display: flex;
+`
+const NavWrapper = styled.div<{ isInfoExplorePageEnabled: boolean }>`
+  display: flex;
+  max-width: ${MAX_WIDTH_MEDIA_BREAKPOINT};
+  margin: 0 auto;
+  margin-bottom: ${({ isInfoExplorePageEnabled }) => (isInfoExplorePageEnabled ? '16px' : '20px')};
+  color: ${({ theme }) => theme.neutral3};
+  flex-direction: row;
+
+  @media only screen and (max-width: ${MEDIUM_MEDIA_BREAKPOINT}) {
+    flex-direction: column;
+    gap: 8px;
+  }
+
+  ${({ isInfoExplorePageEnabled }) =>
```

Comment:
some of the styling on this page gets hard to read bc of the small differences between Tokens & Explore designs. @cbachmeier I know you said readability will take a hit in the meantime but curious what you think on creating a separate file for Explore styled components